lib/kontentClient.ts

```diff
@@ -66,7 +66,7 @@ export const getItemByCodename = <ItemType extends IContentItem>(codename: PerCo
 
 const homepageTypeCodename = "web_spotlight_root" as const;
 
-export const getHomepage = (usePreview: boolean) =>
+export const getHomepage = async (usePreview: boolean) =>
```

Comment:
This is not useful here. The function directly returns a promise, the `async/await` feature doesn't help here.

Reply:
removed unnecessary await